### PR TITLE
Convert to new override syntax

### PR DESCRIPTION
**Pull request by angolini**

### CODE REVIEW

File: recipes-devtools/half/half_2.1.0.bb [hunk 1]
```diff
@@ -17,4 +17,4 @@ do_install () {
     cp -r ${S}/include/* ${D}${includedir}
 }
 
-ALLOW_EMPTY_${PN} = "1"
+ALLOW_EMPTY:${PN} = "1"
```
Comment (thochstein) [inline recipes-devtools/half/half_2.1.0.bb:20]: Is this correct? I'm confused about the overrides with `PN`, like `FILES`, `RDEPENDS`, `PREFERRED_PROVIDER`, and `PREFERRED_VERSION`. I guess the migration docs actually mention that these are tricky but don't really help my understanding...

**Comment by thochstein** [comment]
Here are some more updates: https://github.com/angolini/meta-freescale-distro/pull/1

**Comment by angolini** [comment]
I have included your commits @thochstein  (but haven't had time to build them). Please take a look

**Comment by thochstein** [comment]
Looks good, thanks!

File: recipes-devtools/half/half_2.1.0.bb (hunk 1 above)
Comment (otavio) [reply recipes-devtools/half/half_2.1.0.bb:20]: It is. `${PN}` here is an override.

**Review by otavio — APPROVED** [review]

**Pull request merged**